llama_index/vector_stores/chroma.py

```diff
@@ -129,27 +145,34 @@ def add(self, nodes: List[BaseNode]) -> List[str]:
         if not self._collection:
             raise ValueError("Collection not initialized")
 
-        embeddings = []
-        metadatas = []
-        ids = []
-        documents = []
-        for node in nodes:
-            embeddings.append(node.get_embedding())
-            metadatas.append(
-                node_to_metadata_dict(
-                    node, remove_text=True, flat_metadata=self.flat_metadata
+        max_chunk_size = 41665  # One less than the max chunk size for ChromaDB
```

Comment:
very nit: could we define this as `MAX_CHUNK_SIZE` constant at the top of the file?

Reply:
100% legit, I should ahve done that at the start. Aplogies. Change pushed.